SPECS/python3-geomet/python3-geomet.spec

```diff
@@ -0,0 +1,53 @@
+Name:           python3-geomet
```

Comment:
As a policy, we keep the `Name` field and the spec filename free of specific Python versions- i.e. this should be `python-geomet` and the file path should be `SPECS/python-geomet/python-geomet.spec`. 

(I am aware of some specs that are named similarly, but those have been on my backlog to fix for a while now)

Reply:
Removed the version specific string from names and add a sub-package in SPEC file.